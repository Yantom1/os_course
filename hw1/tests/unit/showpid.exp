smash> smash pid is (?P<pid>\d+)
smash> smash pid is (?P<pid>\d+)
smash> smash> smash pid is (?P<pid>\d+)
smash> smash pid is (?P<pid>\d+)
smash> smash pid is (?P<pid>\d+)
smash> 