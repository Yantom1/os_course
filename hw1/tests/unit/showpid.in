# the test validates that all of the pids are the same using regex groups
showpid
showpid
showpid > my_pid
showpid | grep smash
sleep 1 | showpid
cat my_pid
quit
