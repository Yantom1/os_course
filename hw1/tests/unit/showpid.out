smash> smash pid is 44832
smash> smash pid is 44832
smash> smash> smash pid is 44832
smash> smash pid is 44832
smash> smash pid is 44832
smash> 